api/rest/client_test.go

```diff
@@ -72,27 +72,31 @@ func TestClient_CRUD(t *testing.T) {
 
 		assert.NoError(t, err, "Unexpected error")
 		assert.Equal(t, http.StatusOK, resp.StatusCode, "Expected status code 200")
+		assert.Equal(t, http.MethodGet, resp.RequestInfo.Method)
```

Comment:
if we test that the method is set, shouldn't we the assert the URL string as well?